docs/entities/attributes.md

```diff
@@ -0,0 +1,59 @@
+Attributes
+==========
+
+Attributes are ways to modify specific values for living entities in a mod compatible way. 
+
+Existing Attributes
+-----------
+Vanilla adds the following attributes:
+
+| Attribute                     | Description                                                                                  |
+|-------------------------------|----------------------------------------------------------------------------------------------|
+| `MAX_HEALTH`                  | how much health the entity has                                                               |
+| `FOLLOW_RANGE`                | how far the entity can track something                                                       |
+| `KNOCKBACK_RESISTANCE`        | how strong knockback is reduced. Ranged between 0 and 1, 1 cancelling all knockback.         |
+| `MOVEMENT_SPEED`              | how fast the entity moves                                                                    |
+| `FLYING_SPEED`                | The flying speed of the entity. Not applicable to players who fly with the elytra            |
+| `ATTACK_DAMAGE`               | how much damage the entity deals when attacking                                              |
+| `ATTACK_KNOCKBACK`            | how much knockback the entity causes when physically attacking                               |
+| `ATTACK_SPEED`                | how many full damage attack the entity deals per second                                      |
+| `ARMOR`                       | how many armor points the entity has and how resistant it is to attacks                      |
+| `ARMOR_TOUGHNESS`             | how resistant the entity is to strong attacks                                                |
+| `LUCK`                        | changes the chance for treasure, junk and normal fishing loottables                          |
+| `SPAWN_REINFORCEMENTS_CHANCE` | how likely a zombie spawns reinforcements when attacked                                      |
+| `JUMP_STRENGTH`               | how high the entity jumps. Only applicable for horse like more (including mules and camels). |
```

Comment:
I have no idea how useful this is. This makes sense in parchment, but I feel like this could also just be said it's in the attributes file and list some examples.

Reply:
I do believe that the list of attributes and what they do is useful. There are already parchment comments for them, but giving people examples for what mojang and neo use attributes for, so people can easier understand, what things attributes can be used for and therefore be used for their use cases

Reply:
This is one of those instances where I'm inclined to agree that the docs should delegate to the code for listing the attributes.  This is almost guaranteed to go stale, while a reference to the `Attributes` class will always be accurate.